system/Database/MySQLi/Connection.php

```diff
@@ -98,6 +98,10 @@ public function connect(bool $persistent = false)
         mysqli_report(MYSQLI_REPORT_ALL & ~MYSQLI_REPORT_INDEX);
 
         $this->mysqli->options(MYSQLI_OPT_CONNECT_TIMEOUT, 10);
+        
+        if(isset($this->DBTypeNative) && $this->DBTypeNative){
+            $this->mysqli->options(MYSQLI_OPT_INT_AND_FLOAT_NATIVE,1);
```

Comment:
Please run `composer cs-fix` to fix the coding style issues.

Reply:
Wow ~ I overlooked this, thanks for reminding.